create-budget/azuredeploy.json

```diff
@@ -0,0 +1,160 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "budgetName": {
+      "type": "string",
+      "metadata": {
+        "description": "Budget name"
```

Comment:
What are the naming reqs for this?  i.e. is it globally unique?  State one way or another - add a defaultValue as appropriate

Reply:
Updated the description. Budget name should be unique within the resource group. Adding MyBudget as Default value.